examples/todomvc/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "todomvc"
+version = "0.1.0"
+authors = ["The wasm-bindgen Developers"]
+
+[lib]
+crate-type = ["cdylib"]
+
+[profile.release]
+debug = false
```

Comment:
I believe this is already implied by `profile.release`, so it should be fine to remove

Reply:
This was from: https://rustwasm.github.io/book/game-of-life/code-size.html#disable-debug-symbols

Reply:
Aha! I sent a PR - https://github.com/rustwasm/book/pull/116